### DISCUSSION TITLE
Type expected TS1110 issue on deploying stage

### DISCUSSION THREAD

**Discussion by mchammer1041** [GrapesJS/grapesjs#4742] [Q&A]
I'm getting the following error:

```
/home/user/my_project/node_modules/grapesjs/index.d.ts
TypeScript error in /home/user/my_project/node_modules/grapesjs/index.d.ts(1230,7):
Type expected.  TS1110
    1228 |
    1229 |   type CommandEvent =
  > 1230 |     | run:${string}
         |       ^
    1231 |     | stop:${string}
    1232 |     | abort:${string};
    1233 |
```
    
 I don't know why is this happening since it's only an issue on the prod environment. Unfortunately, the local, testing and prod envinoments are slightly different. I'm pretty sure it's not a GrapesJs issue, but probably someone can give a hint where I should look into

**Comment by artf** [comment]
Maybe something related to the TS version on prod?!
